src/components/ContentOpenWith/OpenWithButton.js

```diff
@@ -29,32 +29,41 @@ type Props = {
  * @private
  * @return {?(string | Element)} the tooltip message
  */
-export const getTooltip = (displayDescription: ?string, error: ?any, isLoading: boolean): ?(string | Element) => {
+export const getTooltip = (
+    displayDescription: string,
+    disabledReasons: Array<DisabledReason> = [],
+    error: ?Error,
+    isLoading: boolean,
+): ?(string | Element) => {
     if (isLoading) {
         return null;
     }
-    if (error) {
-        return <FormattedMessage {...messages.errorOpenWithDescription} />;
-    }
-    if (displayDescription) {
-        return displayDescription;
+
+    let message = <FormattedMessage {...messages.emptyOpenWithDescription} />;
+    if (disabledReasons.length > 0) {
+        [message] = disabledReasons;
+    } else if (error) {
+        message = <FormattedMessage {...messages.errorOpenWithDescription} />;
```

Comment:
This seems like an unnecessary statement, considering it's identical, above

Reply:
What's it identical to? Here we're using the error message but we default to the empty message on L42.